source/ed/osm2ed.cpp

```diff
@@ -338,6 +338,7 @@ void OSMCache::insert_edges() {
             "cycles_allowed", "cars_allowed"});
     std::stringstream geog;
     geog << std::cout.precision(10);
+    std::vector<std::string> coords;
```

Comment:
you can use `navitia::type::LineString` (which is just a `typedef std::vector<GeographicalCoord`). Then, you can use boost to generate the wkt.